### PR TITLE
Add test to check if Barbican secret payload is kept

### PR DESCRIPTION
**Pull request by afaranha**
This is to reflect the workload to adopt step on the documentation.

https://openstack-k8s-operators.github.io/data-plane-adoption/dev/#_creating_a_workload_to_adopt

### CODE REVIEW

File: tests/roles/barbican_adoption/tasks/main.yaml
Comment (xek) [inline tests/roles/barbican_adoption/tasks/main.yaml:6]: Documentation also needs an update

**Comment by afaranha** [comment]
recheck

**Review by vakwetu — COMMENTED** [review]
+2  LGTM

**Review by vakwetu — COMMENTED** [review]
/lgtm

**Comment by afaranha** [comment]
recheck

**Comment by afaranha** [comment]
recheck

**Comment by afaranha** [comment]
recheck

**Review by xek — APPROVED** [review]

File: tests/roles/barbican_adoption/tasks/main.yaml
Comment (marios) [inline tests/roles/barbican_adoption/tasks/main.yaml:81]: why the different in the indentation here (shouldn't matter, but, why?)

**Review by marios — COMMENTED** [review]
having a look here on request (via slack ping).

looks good/has enough approvals, just wondering about the bash indentation (is it intentional for some reason?)

File: tests/roles/barbican_adoption/tasks/main.yaml
Comment (marios) [reply tests/roles/barbican_adoption/tasks/main.yaml:81]: (i can merge once i hear back just waiting for sanity check on this )

Comment (afaranha) [reply tests/roles/barbican_adoption/tasks/main.yaml:81]: My mistake, let me fix it.

Comment (marios) [reply tests/roles/barbican_adoption/tasks/main.yaml:81]: ack thanks ... as just discussed with @xek on slack, since this ran last night if you update just this bit we don't have to wait for a new ci run, I will override

**Pull request merged**
